webapp/packages/plugin-codemirror6/src/Editor.tsx

```diff
@@ -19,10 +19,13 @@ import { useCodemirrorExtensions } from './useCodemirrorExtensions';
 import { type IDefaultExtensions, useEditorDefaultExtensions } from './useEditorDefaultExtensions';
 
 export const Editor = observer<IEditorProps & IDefaultExtensions, IEditorRef>(
-  forwardRef(function Editor({ lineNumbers, extensions, ...rest }, ref) {
+  forwardRef(function Editor({ lineNumbers, extensions, useDefaultExtensions = true, ...rest }, ref) {
     extensions = useCodemirrorExtensions(extensions);
+
     const defaultExtensions = useEditorDefaultExtensions({ lineNumbers });
```

Comment:
It's better to extend the options of useEditorDefaultExtensions to remove unwanted extensions.